packages/babel-plugin-transform-built-ins/src/index.js

```diff
@@ -0,0 +1,128 @@
+"use strict";
+
+const { methods, properties } = require("./builtins");
+const evaluate = require("babel-helper-evaluate-path");
+
+
+
+module.exports = function({ types: t }) {
+
+  class BuiltInReplacer {
+    constructor(program) {
+      this.program = program;
+      this.pathsToUpdate = new Map;
+    }
+
+    run() {
+      this.collect();
+      this.replace();
+    }
+
+    collect() {
+      const context = this;
+
+      const collectVisitor =  {
+        MemberExpression(path) {
+          if (path.parentPath.isCallExpression()) {
+            return;
+          }
+
+          const expName = getExpressionName(path);
+          if (isBuiltin(expName)) {
+            if (!context.pathsToUpdate.has(expName)) {
+              context.pathsToUpdate.set(expName, []);
+            }
+            context.pathsToUpdate.get(expName).push(path);
+          }
+        },
+        CallExpression: {
+          exit(path) {
+            const callee = path.get("callee");
+            if (!callee.isMemberExpression()) {
+              return;
+            }
+
+            const expName = getExpressionName(callee);
+            if (isBuiltin(expName)) {
```

Comment:
should check if it's a computed property

Will fail for `Math[max]()` will assume it as `Math.max()`.